db/seeds.rb

```diff
@@ -12,6 +12,25 @@
              password_confirmation: 'password',
              admin: false
 
+Field.create! name: 'Curb cut',
+              category: 'Accessible',
+              field_type: :choice,
+              #choices: ["Within 20'", 'No curb cut', 'No curb'],
+              description: 'Determine the border between the road and the bus stop.',
+              rank: 2
```

Comment:
I don't see a rank 1 'Accessible' field.

Reply:
? What do you mean.

Reply:
This is a rank 2 field, but I didn't see a rank 1 field.

Reply:
OH WHOOPS my bad